roles/lib_openshift/src/test/integration/filter_plugins/filters.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# vim: expandtab:tabstop=4:shiftwidth=4
+'''
+Custom filters for use in testing
+'''
+
+
+class FilterModule(object):
+    ''' Custom ansible filters '''
+
+    @staticmethod
+    def label_dict_to_key_value_list(label_dict):
```

Comment:
This seems to only be used by the integration tests. Should be add a disclaimer or other annotation in here to indicate this to possible contributors?